pgpy/pgp.py

```diff
@@ -1978,24 +1989,33 @@ def revoke(self, target, **prefs):
         :keyword comment: Defaults to an empty string.
         :type comment: ``str``
         """
+        def _can_revoke(revoker, target_primary):
```

Comment:
why isn't this a normal function of the key. what's wrong with `self.can_revoke(key)`?